render/board.go

```diff
@@ -2,11 +2,39 @@ package render
 
 import (
 	"fmt"
+	"strings"
 
 	"github.com/BattlesnakeOfficial/exporter/engine"
 	log "github.com/sirupsen/logrus"
 )
 
+type snakeCorner string
+
+const (
+	cornerBottomLeft  snakeCorner = "bottom-left"  // ╚
+	cornerBottomRight snakeCorner = "bottom-right" // ╝
+	cornerTopLeft     snakeCorner = "top-left"     // ╔
+	cornerTopRight    snakeCorner = "top-right"    // ╗
+	cornerNone        snakeCorner = "none"
+)
+
+func (c snakeCorner) isBottom() bool {
+	return strings.HasPrefix(string(c), "bottom")
+}
+
+func (c snakeCorner) isLeft() bool {
+	return strings.HasSuffix(string(c), "left")
+}
+
+type snakeDirection int
+
+const (
+	up snakeDirection = iota
+	down
+	left
+	right
```

Comment:
Thinking maybe these should be prefixed, since their names are soooo generic. What about `movingUp`, `movingDown`, etc.?

Reply:
yeah I agree - it was kinda bugging me too